todo_app/data/session_items.py

```diff
@@ -1,22 +1,41 @@
-from flask import session
-
-_DEFAULT_ITEMS = [
-    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
-    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
-]
+from todo_app.data.item import Item
+import requests
+import os
 
 def custom_sort(t):
     return t['status']
 
+def get_lists():
+    """
+    Gets all lists on a board 
+    """
+    params = {
+        'key': os.environ['ACCOUNT_KEY'],
+        'token': os.environ['SECRET_KEY']
+    }
+    response = requests.request('GET', 'https://api.trello.com/1/boards/607f1ee7c99b117ea947851a/lists', params=params)
```

Comment:
Your board id isn't sensitive data, but it is something we'll to want to configure in different environments, so this (and all of the list ids) would be good candidates for also pulling out into environment variables

Reply:
done